package.json

```diff
@@ -9,6 +9,7 @@
     "big.js": "^6.1.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.3.0",
```

Comment:
- [ ] Kindly update the project following [this template](https://github.com/microverseinc/readme-template) and include the live deploy link in the readme.

Reply:
thanks for the review the changes have been made